README.md

```diff
@@ -33,8 +33,8 @@ To make a trait downcastable, make it extend either `downcast::Downcast` or
 `downcast::DowncastSync` and invoke `impl_downcast!` on it as in the examples
 below.
 
-Since 1.1.0, the minimum supported Rust version is 1.33 to support `Rc` and `Arc`
-in the receiver position.
+Since 1.2.0, the minimum supported Rust version is 1.36 due to needing stable
+access to alloc.
```

Comment:
Could you update [the corresponding lines in `lib.rs`][1]? This README is actually generated from that documentation via
```shell
cargo readme >! README.md
```
[1]: https://github.com/marcianx/downcast-rs/blob/17bbb03065aba86905b03484f255119cfc50f993/src/lib.rs#L36-L37